patches/chrome-browser-resources-md_extensions-extensions_resources.grd.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/resources/md_extensions/extensions_resources.grd b/chrome/browser/resources/md_extensions/extensions_resources.grd
+index 7ae0088239c42ce2cca66b28d9f62a8184033983..7fddb06178b5f176c7900c154828d959be6940a4 100644
+--- a/chrome/browser/resources/md_extensions/extensions_resources.grd
++++ b/chrome/browser/resources/md_extensions/extensions_resources.grd
+@@ -161,6 +161,7 @@
+                  type="chrome_html" />
+       <structure name="IDR_MD_EXTENSIONS_SIDEBAR_HTML"
+                  file="sidebar.html"
++                 preprocess="true"
```

Comment:
can you explain this change? I'm not sure what this is doing

Reply:
without preprocess=true, the <if expr> will not be parsed and was shown as <if expr...> in the final HTML.

Reply:
ok, makes sense

Reply:
@yrliou can we pls do a substitution find and replace via the l10n scripts? I prefer not to patch grd files at all.

Reply:
oh, is it possible to change the attributes using that method? That's great if so

Reply:
Ya it's just find and replace, but we can find `<structure name="IDR_MD_EXTENSIONS_SIDEBAR_HTML"` and replace with `<structure name="IDR_MD_EXTENSIONS_SIDEBAR_HTML" preprocess="true"`

```
module.exports.rebaseBraveStringFilesOnChromiumL10nFiles = (path) =>
  Object.entries({
    [chromiumStringsPath]: braveStringsPath,
    [chroimumSettingsPartPath]: braveSettingsPartPath,
    [chromiumComponentsStringsPath]: braveComponentsStringsPath
  }).forEach(([sourcePath, destPath]) =>
    fs.writeFileSync(destPath,
      fs.readFileSync(sourcePath, 'utf8')
        .replace(/settings_chromium_strings.grdp/g, 'settings_brave_strings.grdp')
        .replace(/The Chromium Authors/g, 'Brave Software Inc')
        .replace(/Google Chrome/g, 'Brave')
        .replace(/Chromium/g, 'Brave')
        .replace(/Chrome/g, 'Brave')
        .replace(/Google/g, 'Brave'), 'utf8'))
```